include/swift/AST/DiagnosticsSIL.def

```diff
@@ -245,7 +245,7 @@ WARNING(unreachable_code,none, "will never be executed", ())
 NOTE(unreachable_code_branch,none,
      "condition always evaluates to %select{false|true}0", (bool))
 NOTE(call_to_noreturn_note,none,
-     "a call to a never-returning function", ())
+     "call to never-returning function terminates the program", ())
```

Comment:
Hm. Or infinite-loops. I think this might go against the spirit of diagnostics: only add information if it shows the compiler understands the program; don't add information if it implies the compiler is misunderstanding the program.